game/test/ecs/systems/XPSystemTest.java

```diff
@@ -0,0 +1,73 @@
+package ecs.systems;
+
+import static org.junit.Assert.*;
+
+import ecs.components.xp.ILevelUp;
+import ecs.components.xp.XPComponent;
+import ecs.entities.Entity;
+import mydungeon.ECS;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class XPSystemTest {
+
+    private Entity entity;
+    private XPComponent xpComponent;
+    private XPSystem xpSystem;
+    private ILevelUp levelUp;
+
+    @Before
+    public void setup() {
```

Comment:
Hier kommt schön das issue #213 ins Gespräch. Idealerweise nur `before` oder `after` nutzen, um die Grundfunktionalität zu gewährleisten, alles, was denn Test selbst ausmacht. Z.b. die Komponente soll in die Testmethode. Oder ein super gutes Argument einfach in das Issue schreiben und hier erstmal lassen.

Reply:
Okay, ja habe den before block mal raus genommen.